README.md

```
@@ -258,67 +283,78 @@ Now you can visit [`https://glific.test:4001`](https://glific.test:4001) from yo
 
 Now you can visit [`https://glific.test:4001`](https://glific.test:4001) from your browser.
 
-### 8. Front-end - Install glific front-end
+## 6. Installed fronted
+ [Glific Frontend Repo](https://github.com/glific/glific-frontend)
 
-You cannot do much from the glific backend unless you are an API developer. To see Glific in its glory, please
-install [Glific Frontend](https://github.com/glific/glific-frontend/)
+#### Frontend credentials
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)